api/v2.0/swagger.yaml

```diff
@@ -814,6 +814,148 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
+  /projects/{project_id}/preheat/policies:
+    post:
+      summary: Create a preheat policy under a project
+      description: Create a preheat policy under a project
+      tags:
+        - preheat
+      operationId: CreatePolicy
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/projectId'
+        - name: policy
+          in: body
+          description: The policy schema info
+          required: true
+          schema:
+            $ref: '#/definitions/PreheatPolicy'
+      responses:
+        '201':
+          description: Create policy success
+          schema:
+            $ref: '#/responses/201'
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '409':
+          $ref: '#/responses/409'
+        '500':
+          $ref: '#/responses/500'
+    get:
+      summary: List preheat policies
+      description: List preheat policies
+      tags:
+        - preheat
+      operationId: ListPolicies
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/projectId'
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/pageSize'
+        - $ref: '#/parameters/query'
+      responses:
+        '200':
+          description: List preheat policies success
+          headers:
+            X-Total-Count:
+              description: The total count of tags
+              type: integer
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/PreheatPolicy'
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '500':
+          $ref: '#/responses/500'
+  /projects/{project_id}/preheat/policies/{preheat_policy_id}:
```

Comment:
If we keep the name unique, I think we should also use policy name here to keep consistent with other APIs.